horizons/world/player.py

```diff
@@ -38,7 +38,16 @@
 
 
 class Player(ComponentHolder, WorldObject):
-	"""Class representing a player"""
+	"""Class representing a player
+
+	@type session: horizons.session.Session
+	@param session: Session instance
+	@param worldid: player's worldid
+	@param name: user-chosen name
+	@param color: color of player (as Color)
+	@param clientid: id of client
+	@param inventory: {res: value} that are put in the players inventory
+	"""
```

Comment:
> Probably should look at the rest of the codebase to see if it is really consistent.

I try to find it with:
`git grep -A 1 __init__ | grep -B 1 """` and `git grep -A 1 class | grep -B 1 """`
but this command are not working.